universal-login-wallet/src/index.tsx

```diff
@@ -17,9 +17,7 @@ const AppBootstrapper = () => {
 
     services.walletService.loadFromStorage();
     setBetaNotice(services.sdk);
-
-    const isIncognito = await isPrivateMode();
-    return {services, isIncognito};
+    return {services};
```

Comment:
we can now return services without object wrapping , so we would not have to extracting services after